golem_messages/factories/datastructures/tasks.py

```diff
@@ -16,10 +17,7 @@ class Meta:
         model = dt_tasks.TaskHeader
         exclude = ('requestor_public_key', )
 
-    requestor_public_key = factory.LazyFunction(
-        lambda: encode_key_id(cryptography.ECCx(None).raw_pubkey)
-    )
```

Comment:
This should still be `LazyFunction` to get different pubkeys everytime.

Reply:
done